### PR TITLE
Add home and about page elements style and JS

### PR DESCRIPTION
**Pull request by yash244466666**
Added the following files for elements and style

- [about-app.js](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/about-app.js)
- [about-style.css](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/about-style.css)
- [about.html](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/about.html)
- [ham-app.js](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/ham-app.js)
- [hero-app.js](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/hero-app.js)
- [hero-style.css](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/hero-style.css)
- [index.html](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/index.html)
- [mainProgramSec-app.js](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/mainProgramSec-app.js)
- [mainProgramSec-style.css](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/mainProgramSec-style.css)
- [partner-app.js](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/partner-app.js)
- [partner-style.css](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/partner-style.css)
- [speaker-app.js](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/speaker-app.js)
- [speaker-style.css](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/speaker-style.css)
- [style.css](https://github.com/yash244466666/Microverse-Capstone-One/blob/addElementStyleJs/style.css)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -4,25 +4,106 @@
...
+            </div>
+            <div class="past-summit-img-cont">
+                <div class="past-summit-img1 past-summit-img">
+                    <img src="./res/aboutgroup1.png" alt="past summit img1">
+                </div>
+                <div class="past-summit-img1 past-summit-img">
+                    <img src="./res/aboutgroup2.png" alt="past summit img1">
+                </div>
+                <div class="past-summit-img2"></div>
+            </div>
```
Comment (Qoosim) [inline about.html:81]: - [ ] Your layout for the section of project if different from the layout we have on the template. Kindly follow the layout on the template tp structure your design.

### Template Layout

temp-about-layout


#  Your Layout

your-about-layout

File: about.html [hunk 2]
```diff
@@ -4,25 +4,106 @@
...
+        </section>
+
+
+
+        <section id="partnerDtls" class="partner-dtls">
+            
+
+        </section>
```
Comment (Qoosim) [inline about.html:89]: - [ ] Kindly remove this part for the about page on the desktop. It is not included on the guiding template. Take a look.

### Your Design

your-design


### Template

temp-design

File: about.html [hunk 3]
```diff
@@ -4,25 +4,106 @@
...
+
+        <footer id="footer">
+            <div class="footer-items">
+                <div id="footer-logo">
+                    <img src="./res/footer-logo.svg" alt="logo">
+                </div>
+                <div class="footer-text"><p> &copy; Your project so that you have <br> something unique in your portfolio.</p></div>
+            </div>
+
+        </footer>
```
Comment (Qoosim) [inline about.html:99]: - [ ] The footer for the about page has black background color, but yours is white. Kindly change your background color from white to black

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @yash244466666, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights

-  No linter errors. 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

- [x] There's not video presentation in either your README file or PR. Kindly make a video presentation for your project.

- [x] Kindly look into the text at the left side of your main program section. That text is not supposed to be there. Kindly look into it and fix it. Take a look of what I am saying.

main-program-extral-text

**Comment by yash244466666** [comment]
Added the following changes as instructed 

- [ ]  Added video presentation
- [ ] Fixed the venue time problem
- [ ] Fixed the layout problem on about page
- [ ] Remove the partner section on the about page
- [ ] Fixed the footer section background color as instructed

**Review by sumairq — APPROVED** [review]
### Hi @yash244466666 , 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**